### DISCUSSION TITLE
Add a create new in options.

### DISCUSSION THREAD

**Discussion by pranavmappoli** [excalidraw/excalidraw#8869] [Polls]
<img width="269" alt="Screenshot 2024-12-03 at 1 29 46 PM" src="https://github.com/user-attachments/assets/7e142dbf-a87d-414a-ae24-b5ee25bcc5af">
In the options, its good to have a "new" option, so that we can create a new one

**Comment by Mrazator** [comment]
Yeah, we will likely want to improve this part in the future, let's track it here https://github.com/excalidraw/excalidraw/issues/8572
